java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -178,6 +179,10 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
                 .pfmerge(hllKey3, new String[] {hllKey1, hllKey2})
                 .pfcount(new String[] {hllKey3});
 
+        baseTransaction.bitcount(key3);
+        baseTransaction.bitcount(key3, 2, 4);
+        baseTransaction.bitcount(key3, 2, 19, BitMapOptions.BIT);
```

Comment:
Remove this test - it fails on redis 6.
It is ok to omit it, but if you want to test this in transaction - create a separate test cases for cluster and standalone client. Or wait for https://github.com/aws/glide-for-redis/pull/1284.

Reply:
As discussed, it's better to wait for https://github.com/aws/glide-for-redis/pull/1284. Will add to PR to ignore GHA failures.